src/jupyterhub_config/custom_docker_spawner.py

```diff
@@ -0,0 +1,133 @@
+import os
+import venv
+from pathlib import Path
+
+from dockerspawner import DockerSpawner
+
+
+class CustomDockerSpawner(DockerSpawner):
```

Comment:
it now inherits from the DockerSpawner module. All other funcs are copied from `custom_spawner`